tests/iio_readdev.c

```diff
@@ -249,7 +249,8 @@ int main(int argc, char **argv)
 {
 	unsigned int i, nb_channels;
 	unsigned int buffer_size = SAMPLES_PER_READ;
-	int c, option_index = 0, arg_index = 0, ip_index = 0, uri_index = 0;
+	char *arg_uri = NULL, *arg_ip = NULL;
```

Comment:
nitpick: `const char *` could be added here

Reply:
i checked that the `iio_create_context_` functions do take `const char *` args

Reply:
right, I wanted to change that, but forgot about it, thanks.